openhands/server/routes/manage_conversations.py

```diff
@@ -33,16 +33,12 @@ class InitSessionRequest(BaseModel):
     selected_repository: str | None = None
 
 
-@app.post('/conversations')
-async def new_conversation(request: Request, data: InitSessionRequest):
-    """Initialize a new session or join an existing one.
-    After successful initialization, the client should connect to the WebSocket
-    using the returned conversation ID
-    """
-    logger.info('Initializing new conversation')
-
+async def _create_new_conversation(
```

Comment:
I'd probably move this into a new file, maybe in `opnehands/server/session` (which should probably become `openhands/server/conversation` at this point but no need to do that in this PR)